src/python/base/utils.py

```diff
@@ -948,21 +947,3 @@ def cpu_count():
 
   return environment.get_value('CPU_COUNT_OVERRIDE',
                                multiprocessing.cpu_count())
-
-
-# TODO(mbarbella): Delete this once fully migrated to Python 3.
-def newstr_to_native_str(s):
```

Comment:
Make sure to git grep "newstr_to_native_str" to make sure you removed all, i see some, just want to make sure.